zipline/algorithm.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.algoscript = kwargs.pop('script', None)
 
         self._initialize = None
+        self._analyze = kwargs.pop('analyze', None)
```

Comment:
Most of this test is just from python, so maybe I can get it to work on my
system in albeit some hackish way.

Alternatively, Is there some way to call Travis CI client on my fork of the
code base without requesting a pull?

On Tue, Nov 3, 2015 at 10:34 AM, Richard Frank <notifications@github.com>
wrote:

> In zipline/algorithm.py
> <https://github.com/quantopian/zipline/pull/806#discussion_r43770859>:
>
> > @@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
> >          self.algoscript = kwargs.pop('script', None)
> >
> >          self._initialize = None
> > +        self._analyze = kwargs.pop('analyze', None)
>
> Yep, that's what I would have pointed to you, and sorry our support of
> windows is less than adequate...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quantopian/zipline/pull/806/files#r43770859>.
>